test/Polytopes/types.jl

```diff
@@ -0,0 +1,83 @@
+@testset "types" begin
+
+    a = [1, 2, 3]
+    b = [8, 6, 4]
+
+    for T in [PointVector, RayVector]
+
+        @testset "$T" begin
```

Comment:
Actually testset has a built in looping feature which I'd recommend to use (of course you'll also need to remove one `end` later, and perhaps change other `@testset` uses):
```suggestion
    @testset "$T" for T in [PointVector, RayVector]
```